File: web dev/Java Script/basic/sorting.js
let arr =[9,24,1,87,26,9]
arr = arr.sort();
console.log(arr); // false sorting

brr = arr.sort((a,b) =>{
    return a-b;
})
console.log(brr);

// arr = arr.sort(function(a,b){
//      return b-a;
// });

arr = arr.sort((a,b)=>{
     return b-a;
});

console.log(arr);

File: java/ArrayList/Sorting.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.Scanner;
public class Sorting {
    public static void main(String[] args){

        //Declaration of ArrayList
     ArrayList <Integer>list = new ArrayList<>();

     try(Scanner sc =new Scanner(System.in)){
     
     // prompt the user to enter the size of ArrayList:- 
     System.out.print("Enter size of arraylist:- ");
     int n = sc.nextInt();

     //prompt the user to enter the element of ArrayList:- 
     System.out.println("Enter the element of ArrayList:- ");
     for(int i= 0;i<n;i++){
        list.add(sc.nextInt());
     }

     //sorting in ascending order
     Collections.sort(list);

     System.out.println("in ascending order:- "+ list);

     //for Decending order 
     Collections.sort(list, Collections.reverseOrder());
     System.out.println("In decending order- "+ list);
     
     }
     
    }
    
}
